tools/schemacode/src/bidsschematools/tests/test_utils.py

```diff
@@ -0,0 +1,166 @@
+from contextlib import nullcontext
+from typing import cast
+
+import pytest
+from jsonschema.exceptions import SchemaError, ValidationError
+from jsonschema.protocols import Validator as JsonschemaValidator
+from jsonschema.validators import Draft7Validator, Draft202012Validator
+
+from bidsschematools.utils import jsonschema_validator
+
+
+@pytest.fixture
+def draft7_schema() -> dict:
+    """
+    A minimal valid Draft 7 schema requiring a 'name' property of type 'string'.
+    """
+    return {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "properties": {"name": {"type": "string"}},
+        "required": ["name"],
+    }
```

Comment:
I think these fixtures could just be constants. They're not actually used as fixtures anywhere, right?

Reply:
They are called as fixtures, but you are right, they can be defined as constants, which are better. 

The requested changes are made and pushed.